core/rss/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+    kotlin("jvm")
+}
+
+group = "com.boswelja"
+version = "1.0-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation(project(":core:network"))
```

Comment:
Not sure how I feel about having dependencies on core from core, can we give this responsibility back to Dale? I feel like deserializing responses is specific to bot features

Reply:
Modules can depend on each other. But you are right, some of those should be common, not core

Reply:
About moving back to Dale. We can, but we should have a discussion on the architecture we want to have, define the boundaries and than move on from there

Reply:
Discussion was had, I think we have made agreements on the architecture, right? Can this be closed?